### PR TITLE
1.0.3 Release Process PR

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

**Comment by ebfull** [comment]
ACK

File: .gitignore [hunk 1]
```diff
@@ -65,6 +65,7 @@ src/qt/test/moc*.cpp
 *.a
 *.pb.cc
 *.pb.h
+.vscode
```
Comment (nathan-at-least) [inline .gitignore]: What is this?

Comment (ebfull) [inline .gitignore]: I suggested Simon add it because he was avoiding commiting it, something for his IDE.

**Comment by str4d** [comment]
utACK

File: .gitignore (hunk 1 above)
Comment (bitcartel) [inline .gitignore]: @nathan-at-least https://code.visualstudio.com/

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**